example/install.bash

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# This script is used to run the ansible-galaxy iridl installation.
+# It will check for the existence of the virtual environment: /opt/datalib_venv3.12
+#
+# It will check for any local git changes that might need to be pushed, and it will check for
+# any git changes that need to be pulled.
+##
+VENV=/opt/datalib_venv3.12
+BUILD=""
+CHECK="-diff"
+
+function showhelp()
+{
+  echo "$0 [--build] [--check]"
+  echo " --check:    Optional argument to check what will be done without making changes."
+  echo " --build:    Optional argument to build or rebuild the maprooms.  This can take a long time."
+  echo ""
+  exit
+}
+
+if [ $# -gt 0 ]; then
+  case $1 in
+    --build|-build|build)
```

Comment:
The convention is two hyphens before a long option, one hyphen before a single-letter option, and no hyphens for a positional argument. Don't accept all three, only `--build`. Same comment for `--check` below.